main.py

```diff
@@ -99,13 +100,29 @@ async def on_message(message: Message):
 
 # Take a filename as string and return it formatted nicely
 def format_filename(filename: str):
-    # Remove file extension
-    filename = path.splitext(filename)[0]
 
-    # Replace all underscores with spaces
-    filename = filename.replace("_", " ")
+    # Get all the tags for a track
+    audio = TinyTag.get(
+        str(os.path.join(f"{attachment_directory_filepath}", f"{filename}"))
+    )
+    content = ""
+    # If the tag doesnot exist or is whitespace display the file name only
+    # Otherwise display in the format @user: <Artist-tag> - <Title-tag>
+    if audio.artist is not None and len(audio.artist.strip()) != 0:
```

Comment:
I missed this in my first review, but using `string.isspace()` seems better than checking if `len(string.split()) == 0`
```suggestion
    if audio.artist is not None and not audio.artist.isspace():
```